wiki/Internet_Relay_Chat/en.md

```diff
@@ -1,28 +1,25 @@
----
-needs_cleanup: true
----
-<!-- TODO: awkward terminology, some parts link to dated info, client-specific help should be updated with modern clients -->
-
 # Internet Relay Chat
 
-The [Internet Relay Chat](https://en.wikipedia.org/wiki/Internet_Relay_Chat), also known as IRC, is a well established standardized protocol for chatting with numerous clients available to connect with.
+*For further information, see [Internet Relay Chat (Wikipedia)](https://en.wikipedia.org/wiki/Internet_Relay_Chat "Wikiepdia")*
+
+The **Intenet Relay Chat** (**IRC**) is an application layer protocol for chatting with numerous clients available to connect with. Although not developed by/for osu!, it is an integral part of its infrastructure and to community [bots](/wiki/Bot_account).
```

Comment:
this reads really weirdly.

Reply:
Assuming you mean the ending bit "...and to community bots"?

Reply:
"available to connect with" "Although not developed by/for osu!" "it is an integral part of its infrastructure and to community"